src/Sim.tsx

```diff
@@ -281,12 +330,19 @@ export class Space {
     this.scene.getMeshByName(canName).dispose();
   }
 
-  private buildGround() {
-    this.ground = Babylon.MeshBuilder.CreateGround("mat", { width:118, height:59, subdivisions:2 }, this.scene);
-    this.ground.position.y = -0.8;
-    this.ground.rotate(new Babylon.Vector3(0,1,0),Math.PI / 2);
+  private buildFloor() {
+    this.mat = Babylon.MeshBuilder.CreateGround("mat", { width:118, height:59, subdivisions:2 }, this.scene);
+    this.mat.position.y = -0.8;
+    this.mat.rotate(new Babylon.Vector3(0,1,0),-Math.PI / 2);
+    const matMaterial = new Babylon.StandardMaterial("ground", this.scene);
+    matMaterial.ambientTexture = new Babylon.Texture('static/Surface-A.png',this.scene);
+    this.mat.material = matMaterial;
+    this.mat.physicsImpostor = new Babylon.PhysicsImpostor(this.mat, Babylon.PhysicsImpostor.BoxImpostor,{ mass:0, friction: 1 }, this.scene);
+
+    this.ground = Babylon.MeshBuilder.CreateGround("ground", { width:354, height:354, subdivisions:2 }, this.scene);
+    this.ground.position.y = -0.81;
```

Comment:
At certain camera angles/distances, the mat starts flickering with the floor. Maybe distancing them a bit more would help?

Reply:
Added 0.02 to the distance and it seems to start flickering farther away now though. You should be able to just fit the whole area on the screen now before it start to flicker.